docs/Gemfile.lock

```diff
@@ -38,8 +37,8 @@ GEM
       jekyll (>= 3.5, < 5.0)
       jekyll-sass-converter (>= 1.0, <= 3.0.0, != 2.0.0)
       rubyzip (>= 1.3.0, < 3.0)
-    jekyll-sass-converter (3.0.0)
-      sass-embedded (~> 1.54)
+    jekyll-sass-converter (2.2.0)
+      sassc (> 2.0.1, < 3.0)
```

Comment:
What is the jekyll-sass-converter, and sassc?  I note in all other places we go to a newer version, but here was step ack from 3.0.0 to 2.2.0.

Reply:
They convert Sass to css. This is probably a byproduct of resolving dependencies after running `bundle update rexml`.